### ISSUE TITLE
Description of parameters is not shown for xsd:integer

### ISSUE THREAD

**Issue by MichaelRoeder** [hobbit-project/platform#228]
## Problem

When configuring a benchmark and hovering the field of a parameter, the description for the parameter is shown. However, this does not work for all fields, e.g., for numbers the tooltip is not shown.

**Comment by Ennosigaeon** [comment]
Actually no title is set for any input type during benchmark configuration. Do you mean any other place?

**Comment by MichaelRoeder** [comment]
You are right. It was reported for the integer fields but it was not there for all the fields :+1:

**Comment by romankorf** [comment]
@MichaelRoeder can this be closed? Seems to be merged in v 2.0.1. Can you check this please.
